packages/lb-utils/src/types/annotationView.ts

```diff
@@ -95,6 +95,10 @@ export interface ICuboid extends IDrawingCuboid, IBasicStyle {
   renderEnhance?: (params: IRenderEnhanceParams) => void;
 }
 
+export interface IStaticPoint extends ICoordinate {
+  color: string;
```

Comment:
Don't use the default `color`. I pass `color: red`, will it crash directly? You need to explain that it need `rgba`